docbook/Admin_Guide/en-US/Configuration.xml

```diff
@@ -5,6 +5,57 @@
 <chapter id="admin.config">
 	<title>Configuration</title>
 
+	<para>MantisBT is highly customizable through the web interface and
+		configuration files. Configuration options can be set globally as well
+		as customized for a specific project or user (except options listed in
+		<emphasis>$g_global_settings</emphasis>).
+	</para>
+
+	<para>Configuration options can be set in
+		<emphasis>config_inc.php</emphasis> and in
+		<emphasis>database</emphasis> (using the various manage pages).
+		Values stored in database take precedence over values in
+		<emphasis>config_inc.php</emphasis>. Options stored in database
+		can also be viewed and updated on
+		"Manage > Manage Configuration > Configuration Report" page.
+	</para>
+
+	<para>That's how MantisBT determines which value to use if an option was overridden:</para>
+
+	<orderedlist>
+		<listitem>
+			<para>Look for the specified configuration option in database
+				for current user and current project.
+			</para>
+		</listitem>
+		<listitem>
+			<para>If not found, look for the specified configuration option in
```

Comment:
If so, maybe to add one more phrase before the enumeration like this one:
"If the value is not found at a certain step it goes to the next. Otherwise, stops searching and starts using the found value."
You won't object?